configure.ac

```diff
@@ -807,9 +807,7 @@ export FCFLAGS
 export FFLAGS
 export LDFLAGS
 export LIBS
-export MPILIBNAME
```

Comment:
Doesn't ROMIO configure use this? Or is the information passed in some other way?

Reply:
When we build ROMIO inside MPICH, we always build it as a convenience library and linked directly, right? So I don't see why it needs to know the MPILIBNAME.

Reply:
OK, I've got it now. Makes sense.